kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README

```diff
@@ -0,0 +1,37 @@
+This sample has been modified to create a domain on pv and also to update the domain using the new WDT model.
```

Comment:
What is new WDT Model ?

Reply:
It is the model that contains additional resources or cluster that the domain has to be updated with.